schemas/blocks/introduction.json

```diff
@@ -30,9 +30,13 @@
             },
             "contents": {
               "$ref": "https://eq.ons.gov.uk/common_definitions.json#/contents"
+            },
+            "guidance": {
+              "$ref": "https://eq.ons.gov.uk/common_definitions.json#/introduction_guidance"
             }
           },
-          "additionalProperties": false
+          "additionalProperties": true,
+          "required": ["id", "title"]
```

Comment:
Is title required?. What does runner expect?. Most likely one of, title, guidance or content?.

Reply:
`id` required, the rest part of `anyOf` as discussed. Thanks.